### PR TITLE
Update AUTHORS.rst

### PR DESCRIPTION
**Pull request by lheagy**
Add to the Authors: 

- Xiaolong Wei, @xiaolongw1223
- Santiago Soler, @santisoler
- Nick Williams, @nwilliams-kobold
- John Weis, @johnweis0480
- Kalen Martens, @kalen-sj

#### PR Checklist
* [x] If this is a work in progress PR, set as a Draft PR
* [x] Linted my code according to the [style guides](https://docs.simpeg.xyz/content/getting_started/practices.html#style).
* [x] Added [tests](https://docs.simpeg.xyz/content/getting_started/practices.html#testing) to verify changes to the code.
* [x] Added necessary documentation to any new functions/classes following the
      expect [style](https://docs.simpeg.xyz/content/getting_started/practices.html#documentation).
* [x] Marked as ready for review (if this is was a draft PR), and converted 
      to a Pull Request
* [x] Tagged ``@simpeg/simpeg-developers`` when ready for review.

### CODE REVIEW

**Comment by jcapriot** [comment]
My name is already on there, but looks good. I think in the past we have encourage people to add their own name to the authors list, but we haven't been very loud about that.

**Comment by lheagy** [comment]
Ah, good! I did a quick skim and thought it was strange that I didn't see your name 🙃. But yeah, the rest of these folks have contributed, so it would be great to see them included if they like.

**Comment by santisoler** [comment]
> My name is already on there, but looks good. I think in the past we have encourage people to add their own name to the authors list, but we haven't been very loud about that.

We could add an automatic message that get's triggered after the first PR of a contributor gets merged. We have this in set up in Fatiando:

https://github.com/fatiando/.github/blob/2301c81fabf1f5c7276aaaa89d18e4c6e7de9cd3/.github/config.yml#L15-L21

**Comment by jcapriot** [comment]
Yeah, I like that. Looks like we'll need to add https://github.com/behaviorbot/welcome as a bot.

**Comment by santisoler** [comment]
We are using Probot instead: https://github.com/probot

No preference over anyone. Just noticed that [behaviorbot/welcome](https://github.com/behaviorbot/welcome) hasn't had a commit in ~ 3 years.

**Comment by jcapriot** [comment]
Which probot bot are you using on Fatiando? There seems to be many different one, but I think this is the most relevant, https://probot.github.io/apps/welcome/

**Comment by santisoler** [comment]
> Which probot bot are you using on Fatiando? There seems to be many different one, but I think this is the most relevant, https://probot.github.io/apps/welcome/

Yes, we are using the Welcome one.

File: AUTHORS.rst [hunk 1]
```diff
@@ -18,4 +18,11 @@
 - Thibaut Astic, (`@thast <https://github.com/thast/>`_)
 - Michael Mitchell, (`@micmitch <https://github.com/micmitch/>`_)
 - I-Kang Ding, (`@ikding <https://github.com/ikding/>`_)
-- Richard Scott (`@bluetyson <https://github.com/bluetyson/>`_)
+- Richard Scott, (`@bluetyson <https://github.com/bluetyson/>`_)
+- Joseph Capriotti, (`@jcapriot <https://github.com/jcapriot>`_)
```
Comment (lheagy) [inline AUTHORS.rst:22]:
```suggestion
```

Joe was alread included

**Comment by santisoler** [comment]
After talking about this PR in today's meeting, we think that the people that are being added to the `AUTHORS.md` should give explicit permission to do so before we merge this.

So, please, comment here if you would like to be added to the `AUTHORS.md` file (@xiaolongw1223, @santisoler, @ngodber,  @nwilliams-kobold, @johnweis0480, @kalen-sj).

We plan to merge this PR before the SimPEG v0.21.0 release. Any person that haven't gave their consent won't be added to the `AUTHORS.md`. But if you still want to be added after that, feel free to open a PR and add yourself!

**Comment by nwilliams-kobold** [comment]
Me too please.

**Comment by xiaolongw1223** [comment]
I would like to be added~

**Comment by johnweis0480** [comment]
I would like to be added as well.

**Comment by kalen-sj** [comment]
Santi was kind enough to ask me about this directly. I have no problems, now or in the future, for being on the SimPEG Authors list. My contribution to the code (up to this point) hasn't been close to significant, though. I think it was a few minor edits at most.

**Comment by santisoler** [comment]
I just removed @ngodber from the file because we didn't receive an explicit consent to include their name.

@ngodber you can totally add yourself to the file if that's your desire. Feel free to open a PR for that!

**Pull request merged**